packages/opentelemetry-core/src/trace/spancontext-utils.ts

```diff
@@ -0,0 +1,35 @@
+import { SpanContext } from '@opentelemetry/types';
+
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const INVALID_SPANID = '0';
+export const INVALID_TRACEID = '0';
```

Comment:
Should these be fully padded values? To my knowledge, there aren't many ways a single `'0'` can sneak into a span context.

Reply:
I think the idea was if we are not able to extract the `SpanContext` correctly, then assign INVALID_SPAN_CONTEXT with INVALID_TRACEID(0) and INVALID_SPANID(0).

Just noticed, in the HTTP plugin we are using [_emptySpanContext](https://github.com/open-telemetry/opentelemetry-js/blob/03981e4299282d0611d1255ab0076368d5830753/packages/opentelemetry-plugin-http/src/http.ts#L64), ideally we should have used INVALID_SPAN_CONTEXT there.

/cc @dyladan @obecny